### DISCUSSION TITLE
License of autogen

### DISCUSSION THREAD

**Discussion by TheTechOddBug** [microsoft/autogen#3008] [Q&A]
What type is the autogen license? I can see license-code and license files. The second one shows Creative Common 4.0. Not sure if this is only for docummentation and code has a different license. Not  sure if it is MIT or other. Could someone confirm this point? Thank you very much in advance.

**Comment by Josephrp** [comment]
![image](https://github.com/microsoft/autogen/assets/18212928/2b752280-a9fe-4896-8de3-78939908f180)
it's not supposed to be ambiguous ;-)
